src/inspector_agent.cc

```diff
@@ -712,18 +712,8 @@ bool Agent::Start(const std::string& path,
     return false;
   }
 
-  // TODO(joyeecheung): we should not be using process as a global object
-  // to transport --inspect-brk. Instead, the JS land can get this through
-  // require('internal/options') since it should be set once CLI parsing
-  // is done.
   if (wait_for_connect) {
-    HandleScope scope(parent_env_->isolate());
-    parent_env_->process_object()->DefineOwnProperty(
-        parent_env_->context(),
-        FIXED_ONE_BYTE_STRING(parent_env_->isolate(), "_breakFirstLine"),
-        True(parent_env_->isolate()),
-        static_cast<v8::PropertyAttribute>(v8::ReadOnly | v8::DontEnum))
-        .FromJust();
+    parent_env_->options()->get_debug_options()->break_first_line = true;
```

Comment:
Or maybe we should just leave `process._breakFirstLine` alone in this PR given that it is somewhat special (may be signaled through inspector protocol other than CLI), and only:

1. Set `process._breakNodeFirstLine` in node.js using `getOptionValue('--inspect-brk-node')` - it makes sense because that information only comes from CLI.
2.  Set `process._deprecatedDebugBrk` and `process._invalidDebug`  with `getOptionValue` combinations likewise because those are only detecting whether deprecated CLI flags are passed.

Reply:
I'm not sure... I think it's okay to modify the options object based if the changes reflect the made changes accurately? I can't really tell whether that's happening without taking an in-depth look.

Either way, I think I prefer this over the current approach of using underscored process variables?